src/pages/index.tsx

```diff
@@ -71,35 +26,138 @@ function Home() {
+            <figcaption className="flex flex-col justify-center lg:order-2">
+              <h3>Time Savings</h3>
+              <p>
+                Drill is “feature-on-demand” tool for real-time application profiling that does not affect codebase.
+                It provides the ability to make white box functional.
+              </p>
+            </figcaption>
+          </Figure>
+          <Figure>
+            <img className="md:order-1 lg:order-2" src={useBaseUrl('img/risks-and-t2r.png')} alt="risks and tests to run" />
+            <figcaption className="flex flex-col justify-center md:order-2 lg:order-1">
+              <h3>Risk and Test to Run</h3>
+              <p>Risk is a new or modified uncovered method.
+                Drill4J advises which test you need to run to cover theese Risk methods.
+              </p>
+            </figcaption>
+          </Figure>
+          <Figure>
+            <img className="lg:order-1" src={useBaseUrl('img/coverage.png')} alt="coverage" />
+            <figcaption className="flex flex-col justify-center lg:order-2">
+              <h3>Measure Code Coverage</h3>
+              <p>
+                Code coverage is a tricky metric, but Drill4J provides it with actionable feedback. It allows map tests
+                to code and vice versa.
+              </p>
+            </figcaption>
+          </Figure>
+          <Figure>
+            <img className="md:order-1 lg:order-2" src={useBaseUrl('img/quality-gate.png')} alt="quality gate" />
+            <figcaption className="flex flex-col justify-center md:order-2 lg:order-1">
+              <h3>Quality Gate</h3>
+              <p>
+                You can configure 3 key metrics: Code Coverage, Risks and Tests to Run
+                and use Drill4J Quality Gate status (passed, failed) in a CI/CD pipelines.
+              </p>
+            </figcaption>
+          </Figure>
+          <Figure>
+            <img className="lg:order-1" src={useBaseUrl('img/browser-extension.png')} alt="browser extension" />
+            <figcaption className="flex flex-col justify-center lg:order-2">
+              <h3>Chrome Browser Extension</h3>
+              <p className="mb-8 lg:mb-0">
+                Browser extension allows gather manual Code Coverage for several users
+                on the one test environment at the same time.
+              </p>
+            </figcaption>
+          </Figure>
+        </section>
+        <section className={clsx(styles.integrateStepsWrapper, 'py-18 text-center')}>
+          <div className="container flex flex-col justify-center">
+            <h2 className="mb-12">Easy to integrate</h2>
+            <ol className="grid lg:grid-cols-4 md:grid-cols-2 gap-x-12 gap-y-8 p-0 mb-8 md:mb-20">
+              <li>
+                <span className={clsx(styles.integrateStepNumber, styles.bgBlueGradient)}>1</span>
+                <h4 className="mb-2">Run Drill4J</h4>
```

Comment:
We should not skip heading ranks to preserve accessibility. In that case `<h4>` tag should not be used, because there is no `<h3>` tag. [See the reference](https://www.w3.org/WAI/tutorials/page-structure/headings/#?:~:text=skipping%20heading%20ranks%20can%20be%20confusing%20and%20should%20be%20avoided%20where%20possible)
I suggest changing that element to `<h3>`

Reply:
The motivation: skipped heading ranks break the outline of the page. You cannot read the page as the document anymore.
In the perfect case, you should be able to delete all styles (_e.g. just delete `<head>` tag for a quick check_) and still be able to make sense of what's on the page.